### DISCUSSION TITLE
how to apply custom font for all slides?

### DISCUSSION THREAD

**Discussion by zzzz465** [marp-team/marp#353] [Q&A] [answered]
Hi, I'm trying to set a custom font for all my slides. I copied `uncover.scss` from [github](https://github.com/marp-team/marp-core/blob/main/themes/uncover.scss) and tried to change `font-family` attributes to the desired font. but it seems not working.

I manually override css properties on top of the YAML file, then it works fine.
did I miss something?

uncover.scss modify
![image](https://user-images.githubusercontent.com/46273764/190957252-14758c8b-98bc-4a61-b27f-60ce9a11c543.png)

overriding css properties
![image](https://user-images.githubusercontent.com/46273764/190957289-1580280e-af39-4b80-bda7-151016f6f004.png)

note that adding `!important` keyword in `uncover.scss` doesn't help.

**Comment by yhatt** [comment, marked as answer]
Note [the hosted source code of themes in marp-team/marp-core](https://github.com/marp-team/marp-core/blob/main/themes) are [Sass](https://sass-lang.com/) (`.scss`), not CSS (`.css`). Both of Marp and browser can recognize only CSS. If Sass syntax was included in the CSS theme, styles may not apply correctly due to syntax error.

Before using in Marp tools, you would have to compile Sass syntax to CSS.

> If you think that's too hard, just [importing pre-compiled theme by `@import`](https://marpit.marp.app/theme-css?id=customized-theme) would help you.
>
> ```css
> /* @theme your-theme */
>
> @import 'uncover';
>
> section {
>   font-family: cursive;
> }
> ```
